manager/manager.go

```diff
@@ -541,10 +541,11 @@ func (m *Manager) Run(parent context.Context) error {
 
 	close(m.started)
 
+	raftErrCh := make(chan error, 1)
 	go func() {
-		err := m.raftNode.Run(ctx)
-		if err != nil {
+		if err := m.raftNode.Run(ctx); err != nil {
 			log.G(ctx).WithError(err).Error("raft node stopped")
+			raftErrCh <- err
```

Comment:
defer a close on this channel so that waiters don't block if this never returns an error.